src/js/vic-v2/components/CropperController.jsx

```diff
@@ -1,10 +1,12 @@
+import _ from 'lodash';
 import React from 'react';
 import Cropper from 'react-cropper';
 import classNames from 'classnames';
 
 const MIN_SIZE = 350;
 const SMALL_CROP_BOX_SIZE = 240;
 const LARGE_CROP_BOX_SIZE = 300;
+const LARGE_SCREEN = 1201;
```

Comment:
I'm seeing an issue where resizing the screen above this size doesn't correctly set the crop box size.

Reply:
![resizing](https://user-images.githubusercontent.com/4998130/37408635-7ea2d4ca-276a-11e8-981a-a58d726dc5b6.gif)

This is what I'm seeing locally. Remounting the canvas is debounced so it won't be smooth as the CSS driven changes. I could force it to re-mount when it crosses that threshold.

Reply:
What I was seeing was that if I resized while staying above 1200 (so like 1205->1300 and vice versa) it was not resetting correctly.

Reply:
I recreated this issue on the review page. Looking into it.

Reply:
This should be repaired now.